server/models/campaign.py

```diff
@@ -166,6 +167,20 @@ def as_dict(self):
             self, recurse=False, exclude=[Note.room, Note.location, Note.user]
         )
 
+class Marker(BaseModel):
+    from .shape import Shape
+
+    uuid = ForeignKeyField(Shape, backref="markers", on_delete="CASCADE")
+    user = ForeignKeyField(User, backref="markers", on_delete="CASCADE")
+    room = ForeignKeyField(Room, backref="markers", on_delete="CASCADE")
```

Comment:
Any particular reason why you chose to use Room and not Location?

Reply:
Nope. I started originally with a copy of Note because it looked similar to what I wanted. After a lot of messing about with a lot a files, the result is completely different :-) 
I've changed it to reference location instead of room.